src/sonic-ctrmgrd/tests/ctrmgr_tools_test.py

```diff
@@ -0,0 +1,290 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from . import common_test
+import sys
+import os
+sys.path.append("ctrmgr")
+import ctrmgr_tools
```

Comment:
Please reorganize as per other similar comments.

Reply:
I bet I did. Not sure how I lost it. I will make sure to get it correct, this round. Sorry about it